mteb/tasks/Classification/hin/HindiDiscourseClassification.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class HindiDiscourseClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="HindiDiscourseClassification",
+        dataset={
+            "path": "hindi_discourse",
+            "revision": "218ce687943a0da435d6d62751a4ab216be6cd40",
+        },
+        description="A Hindi Discourse dataset in Hindi with values for coherence.",
+        reference="https://aclanthology.org/2020.lrec-1.149/",
+        type="Classification",
+        category="s2s",
+        eval_splits=["train"],
+        eval_langs=["hin"],
```

Comment:
You will need to specify the script as well. Otherwise wonderful metadata!

Reply:
Done